GPU/Common/GPUStateUtils.h

```diff
@@ -207,7 +207,6 @@ inline bool SpongebobDepthInverseConditions(const GenericStencilFuncState &stenc
 		gstate.getDepthTestFunction() == GE_COMP_GEQUAL &&
 		stencilState.zFail == GE_STENCILOP_ZERO && stencilState.sFail == GE_STENCILOP_KEEP && stencilState.zPass == GE_STENCILOP_KEEP &&
 		stencilState.testFunc == GE_COMP_ALWAYS &&
-		stencilState.writeMask == 0xFF && stencilState.testMask == 0xFF && stencilState.testRef == 0xFF &&
```

Comment:
`writeMask` should matter, even for 0?

Also, `gstate.getMaterialAmbientA() == 0x0` depends on the material update flag right?  It seems like just checking these values might still be unsafe, although this is probably a pretty narrow case at this point.

-[Unknown]